src/tootsee.cr

```diff
@@ -44,12 +44,19 @@ module Tootsee
 
     # Let's go 🎉
     listener.listen do |mention|
-      puts("Received mention: #{mention}")
-      image = imager.image(mention[:text])
-      puts("Image: #{image}")
-      caption = captioner.caption(image)
-      puts("Caption: #{caption}")
-      replier.reply(caption, mention)
+      spawn do
+        puts("Received mention: #{mention}")
+        image = imager.image(mention[:text])
+        puts("Image: #{image}")
+        caption = captioner.caption(image)
+        puts("Caption: #{caption}")
+
+        begin
```

Comment:
Let's put the `rescue` inside of `replier.reply` so we can keep the main loop clean :3

Reply:
Good idea (: